cloudbuild-test.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 Google LLC
```

Comment:
Does this need to be on the repo root? I don't mind that much but if it can be in a folder similar to the actions folder that'd be good, like `.cloudbuild`